include/ruby/internal/encoding/encoding.h

```diff
@@ -1055,6 +1055,9 @@ RBIMPL_SYMBOL_EXPORT_END()
 #define rb_enc_right_char_head   rb_enc_right_char_head
 #define rb_enc_step_back         rb_enc_step_back
 #define rb_enc_str_asciicompat_p rb_enc_str_asciicompat_p
+#define is_usascii_enc(enc) (enc) == rb_usascii_encoding()
+#define is_ascii8bit_enc(enc) (enc) == rb_ascii8bit_encoding()
+#define is_locale_enc(enc) (enc) == rb_locale_encoding()
```

Comment:
Since these macros seem not used under `ext` or `include`, isn't `internal/encoding.h` better?
Or, should be prefixed at least.

Reply:
Thanks!
Added `rb_` prefix and move to `internal/encoding.h` in https://github.com/ruby/ruby/pull/6700/commits/f29a25ac1f3b0af8b8aa29a523c6f51a4b427188